src/components/backup/list.vue

```diff
@@ -17,7 +17,7 @@ except according to the terms contained in the LICENSE file.
         @terminate="showModal('terminate')"/>
       <page-section v-if="audits.length !== 0">
         <template #heading>
-          <span>Latest Backups</span>
+          <span>{{ $t('auditsTitle') }}</span>
```

Comment:
Do you maybe mean `backupsTitle`?

Reply:
Ah interesting. This is the section at the bottom of the page that lists audit log entries, which is why I chose that key. But of course, they're also all backup-related audit log entries. `BackupStatus` is what shows the main title for the page, so I think I'd lean toward keeping `auditsTitle` to try to make that distinction clearer.

Reply:
That makes sense!